extensions/objects/templates/FilepathEditor.js

```diff
@@ -217,10 +222,10 @@ oppia.directive('filepathEditor', [
           var ctx = canvas.getContext('2d');
           ctx.drawImage(img, 0, 0, dimensions.width, dimensions.height);
 
-          // Return a File object obtained from the data in the canvas.
-          var file = $scope.data.metadata.uploadedFile;
-          var blob = dataURIToBlob(canvas.toDataURL(file.type, 1));
-
+          // We export to PNG because not all browsers support exports to all
+          // possible file types (like gif).
```

Comment:
This comment seems redundant since it was also done in the definition of the constant. Another reason not to have this comment is that, it says 'PNG' but when you look at the code all you see is RESAMPLED_IMAGE_FORMAT so it's hard (for the reader) to link the comment to the code.

I would suggest one of two things, either remove the comment (we have it already at the definition of the constant) or keep the comment but remove the constant (hardcode the 'image/png' here, which will go along with the comment right there).

Wdyt?